app/models.py

```diff
@@ -12,7 +14,18 @@ class Plant(db.Model):
     root_depth = db.Column(db.Integer)
     annual = db.Column(db.Boolean)
 
+    def in_garden(self):
+        Garden.query.filter_by(plant_id=self.id).first()
```

Comment:
If this were being scaled up, we would probably want to reference the user id instead of just finding the first garden in the list.